server/src/infra/auth_service.rs

```diff
@@ -508,6 +659,7 @@ where
             web::resource("/opaque/login/finish")
                 .route(web::post().to(opaque_login_finish::<Backend>)),
         )
+        .service(web::resource("/login").route(web::post().to(conventional_login::<Backend>)))
```

Comment:
Maybe `/login/simple`?

Reply:
I changed the routes to `/simple/login` and `/simple/register`